ambf_ros_modules/ambf_comm/scripts/ambf_rigid_body.py

```diff
@@ -580,6 +580,37 @@ def set_joint_effort(self, joint, effort):
 
         self._apply_command()
 
+    def set_multiple_joint_effort(self, efforts, index_list=[]):
+        """
+        Set the seletecd joint efforts
+        :param q_list: list of the joints to set
+        :return:
+        """
+        n_jnts = len(self._state.joint_positions)
+
+        if len(index_list) == 0:
+            index_list = list(range(n_jnts))
+
+        elif max(index_list)+1 > n_jnts or min(index_list) < 0:
+            print('Requested Joint index is out of range with joints ' + str(n_jnts))
+            return
+
+        elif len(efforts) != len(index_list):
```

Comment:
This ELIF doesn't make sense here and one can easily imagine a scenario where it would not trigger due to prior two conditions. This should be added as an OR statement in the first IF statement.